plugins/scaffolder/src/components/hooks/useEventStream.ts

```diff
@@ -100,7 +102,9 @@ function reducer(draft: TaskStream, action: ReducerAction) {
           }
 
           if (
-            ['cancelled', 'failed', 'completed'].includes(currentStep.status)
+            ['aborted', 'cancelled', 'completed', 'failed'].includes(
```

Comment:
makes me wonder what the distinction is between cancelled and aborted 🤔

Reply:
I can use 'cancelled', but then we can also rename the action to "Cancel", not "Abort" :) to avoid confusion later.
Seems like 'cancelled' is not used so far

Reply:
Cancel is a pretty good word. Abort has some negative connotations and used less in regular speech.

Reply:
Okie dok. I'll change all to "Cancel".
I can't run backstage anymore after merged master to my branch.

Will wait when it be fixed first.
Seems like something is wrong with transpiler configuration for this plugin.

<img width="1047" alt="Screenshot 2023-02-22 at 21 52 22" src="https://user-images.githubusercontent.com/2265094/220756403-bad88a0d-b06e-4133-9b80-bea6fcccb239.png">